spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let(:user){FactoryBot.create :user}
```

Comment:
phân biệt cho c let và let!

Reply:
let sử dụng lazy evaluation nghĩa là let chỉ tạo dữ liệu khi được gọi lần đầu tiên và có giá trị không đổi trong các lần gọi tới.
let! sử dụng để ép buộc gọi trước khi example (các khối describe, context, it) được thực thi, dùng khi muốn lấy dữ liệu trạng thái object trước khi thực hiện hành động nào đó.